.travis.yml

```diff
@@ -50,7 +51,9 @@ install:
 
   # Conda debug
   # -----------
-  - conda list
+  - conda list -n $ENV_NAME
```

Comment:
No need to do both. Happy with the export option though! 👍

Reply:
The one is more for humans, as `conda list` keeps the packages in alphabetical order... otherwise I've found it difficult to quickly eyeball a specific package within the explicit list (which might be in dependency order? and so not too helpful to us humans)

That said, your call...